src/modules/frames/text_frame.c

```diff
@@ -37,13 +37,19 @@ ID3v2_TextFrame* TextFrame_parse(CharStream* frame_cs, const int id3_major_versi
     CharStream_seek(frame_cs, ID3v2_FRAME_ENCODING_LENGTH, SEEK_CUR); // skip encoding
 
     const int text_size = header->size - ID3v2_FRAME_ENCODING_LENGTH;
-    char text[text_size];
+    const size_t string_termination_bytes = 2;
+    char* text = malloc((string_termination_bytes + text_size) * sizeof(char));
```

Comment:
It's been a while since I've worked on this, but, doesn't `text_size` already include the string_termination_bytes length?

Reply:
Based on mp3's I've worked with, there's no termination, so it's not included